src/frontend/js/hooks/useJoanieUserAbilities/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { isJoanieEnabled } from 'api/joanie';
+import { useJoanieUserProfile } from 'hooks/useJoanieUserProfile';
+import AbilitiesHelper from 'utils/AbilitiesHelper';
+
+export const useJoanieUserAbilities = () => {
+  if (!isJoanieEnabled) {
+    return AbilitiesHelper.buildEntityInterface({
+      id: 'ANONYMOUS',
+      username: 'anonymous_user',
+      full_name: 'Anonymous user',
+      is_superuser: false,
+      is_staff: false,
+      abilities: {},
+    });
```

Comment:
We could just return `undefined` WDYT?

```
undefined === "Not configured"
null === "Fetching..."
Object === "Fetched"
```

Reply:
I prefer to have a anonymous object.
but i'll change it if you don't like it this way.